### ISSUE TITLE
Deserialize Python Object

### ISSUE THREAD

**Issue by king0ffire** [CSCI-GA-2820-FA24-003/inventory#16]
**As a** developer in scrum team
**I need** to translate JSON message into python object 
**So that** python can handle JSON requests from RESTful API
   
 ### Details and Assumptions
 * JSON is received as part of RESTful API requests.
   
 ### Acceptance Criteria  
   
 ```gherkin
 Given a JSON object that includes fields like 'id' and 'name'
 When the function is called
 Then it returns a Python object that corresponds to the JSON structure
```

**Comment by egnechng** [comment]
I think we should be more specific on the title. For example, "Deserializing Python Objects"

**Comment by king0ffire** [comment]
Implemented in #20
